### ISSUE TITLE
Clickables: Links cannot be empty

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#75]
**Requirement:**
Links should include text that is setting expectations to what page they are leading to. If empty links are used as layout wrapping elements, a screen-reader only text or an "aria-label" attribute should be used for that description. 

**10 Failed Elements**

- [x] 1.
```html
<a class="navbar-item navbar-logo" href="/" > <img class="navbar-logo_img" src="/assets/images/beaty-logo-horizontal-white.png" height="28"> </a>
```

- [x] 2.
```html
<a href="/specimen/search?country=CA" target="_blank" class="feature-overlay" ></a>
```

- [x] 3.
```html
<a href="/specimen/search?view=MAP" target="_blank" class="feature-overlay" ></a> 
```

- [x] 4.
```html
<a href="/about" target="_blank" class="feature-overlay" ></a>
```

- [x] 5.
```html
<a href="/specimen/search?mediaType=StillImage&amp;view=GALLERY" target="_blank" class="feature-overlay" ></a>
```
- [x] 6.
```html
<a href="/collection/8f5f5b6f-28c6-44b4-8f21-98c55eaae203" class="feature-overlay" ></a>
```

- [x] 7.
```html
<a href="/collection/3b2ad644-b3e4-4ac9-a57f-23be3f86ed0e" class="feature-overlay" ></a>
```

- [x] 8.
```html
<a href="/collection/b44fcb7f-1227-4fa3-8ed2-de27aabb06e0" class="feature-overlay" ></a>
```

- [x] 9.
```html
<a href="https://beatymuseum.ubc.ca" ><img src="/assets/images/beaty-logo-horizontal.png"></a>
```

- [x] 10.
```html
<a href="https://gbif.org/hosted-portals" ><img src="/assets/theme/img/hostedby.svg"></a>
```
